scl/paloalto/panos.conf

```diff
@@ -0,0 +1,97 @@
+#############################################################################
+# Copyright (c) 2020 Balabit
+# created by mileek@gmail.com
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 as published
+# by the Free Software Foundation, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+#
+# As an additional exemption you are allowed to compile & link against the
+# OpenSSL libraries as published by the OpenSSL project. See the file
+# COPYING for details.
+#
+#############################################################################
```

Comment:
A few conditional parses are happening in this file. Could you provide a sample input as comments here, so that we can write a test for it?

Or, if you want to try to write a test yourself, you can get some ideas from https://github.com/syslog-ng/syslog-ng/pull/3056/

Reply:
Currently I can only provide log for two cases:

SYSTEM message:
`<12>Apr 14 16:48:54 paloalto.test.net 1,2020/04/14 16:48:54,unknown,SYSTEM,auth,0,2020/04/14 16:48:54,,auth-fail,,0,0,general,medium,failed authentication for user \'admin\'.  Reason: Invalid username/password. From: 10.0.10.55.,1718,0x0,0,0,0,0,,paloalto`

CONFIG message:
`<14>Apr 14 16:54:18 paloalto.test.net 1,2020/04/14 16:54:18,unknown,CONFIG,0,0,2020/04/14 16:54:18,10.0.10.55,,set,admin,Web,Succeeded, deviceconfig system,127,0x0,0,0,0,0,,paloalto`

Reply:
That's nice, even if not complete coverage. Please add these two as comment, [similarly](https://github.com/syslog-ng/syslog-ng/blob/06df1d712cf0608845eba0662e2cba7f85703af8/scl/cisco/plugin.conf#L33) to the other scl-s.